src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static ArrayList<Task> taskList = new ArrayList<>();
+
+    public static void main(String[] args) throws DukeException {
+        System.out.println("Hi there, I'm TARS!\nWhat can I do for you?");
+        ReadWriteFile.readTaskList(taskList);
+//        for (Task t : taskList) {
+//            System.out.println(t);
+//        }
+        Scanner scanner = new Scanner(System.in);
+        boolean stop = false;
```

Comment:
According to the Java coding standard guide, under the Naming section, it says that boolean variables/method should sound like booleans. Should this be `boolean isStop` or `boolean shouldStop`  instead of `boolean stop`?

Reply:
This makes sense, thanks!